src/jaegertracing/TracerTest.cpp

```diff
@@ -250,6 +249,10 @@ TEST(Tracer, testTracer)
 
     span.reset();
 
+    // There is a problem here. After replacing the global tracer, the spans produced
+    // hold the last references to the tracer. After they are published, the tracer is
+    // destroyed. This is not the right moment to close the tracer.
+    tracer->Close();
```

Comment:
Why do you believe that is an issue? Who cares when the old tracer is closed?

Reply:
The problem is that the tracer is being destroyed. During that phase; the tracer will close the resources that are still used to report the the spans asynchronously.

Reply:
If you see a crash, please report it as a separate issue. I would like to research this a little more. The real fix should never involve an explicit close. For now, please remove these fixes and we can rebase this on top of another PR for this issue.

Reply:
Ok I see. In fact, the point was not fix the crash but to avoid having the crash until it's fixed.

I will deactivate this test for the moment under windows and report it as an issue with the details.

Reply:
Sure, but I want to try fixing that bug first. If it takes more than a day or two, we can disable the test to land this PR.

Reply:
Issue reported #127

Reply:
similarly, add link to the issue in the comments under TODO: